content/concepts/consensus-network/federated-sidechains.md

```diff
@@ -68,8 +66,8 @@ Setting up a sidechain involves the following high-level steps:
     - [sidechain_federators] stanza - list of federators public keys to be used for signing. This list is common for all federators on the sidechain.
 4. Set up door accounts to enable cross chain transactions. This involves the following steps (on _both_ chains):
     - Create and fund the door accounts. 
-    - Set the signers list for the door accounts.
-    - Reserve 3 [tickets](tickets.html) for error handling.
+    - Set the ["SignerList"](set-up-multi-signing.html) for door accounts.
+    - Reserve three(3) [tickets](tickets.html) for error handling.
```

Comment:
```suggestion
    - Create three [tickets](tickets.html) for error handling.
```

I suggest "Create" instead of "Reserve" since the transaction type is TicketCreate and the "Reserve" is something different.

As for the numeral or spelled-out number thing, the Microsoft style guide has this to say:

> In body text, use numerals for 10 and greater. Spell out zero through nine if the number does not precede a unit of measure or is not used as input. [Good examples: 3 centimeters, five databases; bad examples: 2 disks, twelve functions].

You could argue that "3" here is going to be used as input to a TicketCreate transaction but it's indirect enough here that I think spelling it out is appropriate. I would be OK with either one, but doing both with the parentheses here is almost certainly more confusing.